k8s/charts/openebs/README.md

```diff
@@ -1,22 +1,55 @@
+------------------------------------------------------------------------------
+IMPORTANT!!
+
+DEPRECATION NOTICE:
+
+The support for this chart will be discontinued soon. Please plan to migrate
+and use stable/openebs chart located at:
+ [https://github.com/kubernetes/charts/tree/master/stable/openebs](https://github.com/kubernetes/charts/tree/master/stable/openebs)
+
+------------------------------------------------------------------------------
 
 ## Prerequisites
 - Kubernetes 1.7.5+ with RBAC enabled
 - iSCSI PV support in the underlying infrastructure
+- helm is installed and the tiller has admin privileges. To assign admin
+  to tiller, login as admin and use the following instructions:
+  ```
+  kubectl -n kube-system create sa tiller
+  kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+  kubectl -n kube-system patch deploy/tiller-deploy -p '{"spec": {"template": {"spec": {"serviceAccountName": "tiller"}}}}'
+  kubectl -n kube-system patch deployment tiller-deploy -p '{"spec": {"template": {"spec": {"automountServiceAccountToken": true}}}}'
```

Comment:
@kmova , would it be good to change both these spec changes.strategic merge patch as part of the same patch deployment step instead of doing two separate steps.

Reply:
It appears that between releases, different kinds of patches need to be applied. The last one was required from 2.9.0 helm release. These are steps taken from the work-around specified in this bug https://github.com/kubernetes/helm/issues/3985. 

In the future, the helm will not be dependent on tiller and we will not have to perform these steps.